src/test/java/com/eppo/sdk/EppoClientTest.java

```diff
@@ -116,24 +117,19 @@ public AssignmentValueType deserialize(JsonParser jsonParser, DeserializationCon
 
   @BeforeEach
   void init() {
-    setupMockRacServer();
+    setupMockRacServer("src/test/resources/rac-experiments-v3.json");
```

Comment:
Parameterized the mocking of the RAC server so that we can point it at different files for different tests.